src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -27,6 +28,18 @@ class Rule:
     src_table: str
     dst_table: str
 
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]):
```

Comment:
Why do we need it?

Reply:
`table_mapping_mock` uses `cls.from_dict(_)` to load a dataclass from a json